source/overview/security.rst

```diff
@@ -2,74 +2,133 @@
+memory store size, 
+remote address,
+and headers.
+
+Session length, 
+session cache,
+and idle timeout can be `configured according to your internal policies <https://docs.mattermost.com/administration/config-settings.html#sessions>`_. This forces users to login again after a specified period of time.
+
+Remotely `revoke user sessions <https://docs.mattermost.com/help/settings/account-settings.html#view-and-logout-of-active-sessions>`_ across web, 
+mobile devices,
+and native desktop applications
+and remotely reset user passwords through the System Console or through the `command line <https://docs.mattermost.com/administration/command-line-tools.html#platform-user-password>`_.
+
+Mattermost supports integrated authentication with `Active Directory and LDAP <https://docs.mattermost.com/deployment/sso-ldap.html>`_ on Mattermost Enterprise Editions E10 and E20,
+and `SAML 2.0 SSO integration <https://docs.mattermost.com/deployment/sso-saml.html>`_ with providers including `Active Directory Federation Services <https://docs.mattermost.com/deployment/sso-saml-adfs.html>`_,  `Okta <https://docs.mattermost.com/deployment/sso-saml-okta.html>`_, among others on Mattermost Enterprise Edition E20.
+
+Users can mandate `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`_ on Mattermost Enterprise Editions E10 and E20.
 
 Access Control Policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - Limit communications to specific users, private channels, or team-wide public channels
-   - Increase system security `by restricting email-based account creation to email addresses from a list of specific domains, <https://docs.mattermost.com/administration/config-settings.html#restrict-account-creation-to-specified-email-domains>`_ e.g. "corp.mattermost.com", "mattermost.org", etc."
-   - Choose whether to restrict or enable `cross-origin requests. <https://docs.mattermost.com/administration/config-settings.html#enable-cross-origin-requests-from>`_
-   - If sharing of public links for account creation or sharing of files and images are enabled, links can be invalidated via the System Console by `regenerating salts <https://docs.mattermost.com/administration/config-settings.html#public-link-salt>`_.
-   - Optionally restrict `creation, renaming, archiving of channels, private channels and integrations to team admins, system admins or end users <https://docs.mattermost.com/administration/config-settings.html#policy-enterprise>`_ (E10)
-   - Optionally restrict `sending team invites to team admins, system admins or end users <https://docs.mattermost.com/administration/config-settings.html#policy-enterprise>`_ (E10)
-   - Optionally add `advanced passwords requirements <https://docs.mattermost.com/administration/config-settings.html#password-requirements-enterprise>`_ with minimum numbers of symbols, numbers, and lower and uppercase letters (E10).
+All of Mattermost's editions can limit communications to specific users, 
```

Comment:
Need to check this sentence for accuracy, I'm not actually sure what old doc was trying to say

Reply:
okay, I will leave that to you...